module/metrics/consensus.go

```diff
@@ -83,22 +70,18 @@ func NewConsensusCollector(tracer module.Tracer, registerer prometheus.Registere
 
 // StartCollectionToFinalized reports Metrics C1: Collection Received by CCL→ Collection Included in Finalized Block
 func (cc *ConsensusCollector) StartCollectionToFinalized(collectionID flow.Identifier) {
-	cc.tracer.StartSpan(collectionID, consensusCollectionToFinalized)
 }
 
 // FinishCollectionToFinalized reports Metrics C1: Collection Received by CCL→ Collection Included in Finalized Block
 func (cc *ConsensusCollector) FinishCollectionToFinalized(collectionID flow.Identifier) {
```

Comment:
None of the implementation of these methods have a body now. Should they be removed? or is this something that will be used in the future?

Reply:
but still has to satisfy the requirements for ConsensusMetrics interface